src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -63,6 +63,10 @@ protected function has(string $id): bool
      */
     protected function get(string $id)
     {
+        if ('message_bus' === $id) {
+            @trigger_error(sprintf('The "%s" service id is deprecated. Use the "%s" service id instead.', $id, 'messenger.default_bus'), E_USER_DEPRECATED);
+        }
+
```

Comment:
This change can be reverted once the alias is marked as deprecated (see my previous comment)